.vscode/settings.json

```diff
@@ -0,0 +1,64 @@
+{
+  // If the following is ever addressed, we can update our config to use whatever
+  // the official shared config is:
+  // https://github.com/microsoft/vscode/issues/40233
+
+  // This is required to load extensions, which in turn is required to be able
+  // to format SQL files.
+  // https://github.com/prettier/prettier-vscode/issues/3235
+  "prettier.configPath": "./.prettierrc.json",
+
+  "[python]": {
+    "editor.defaultFormatter": "ms-python.black-formatter",
+    "editor.formatOnSave": true,
+    "editor.codeActionsOnSave": {
+      "source.organizeImports": "explicit"
+    }
```

Comment:
Most of the configs I see add an argument to the isort command: https://stackoverflow.com/a/72667901

Reply:
Import sorting works fine for me without this. Doesn't `isort` pick up any needed config from `pyproject.toml`?

Reply:
There are only some cases where black and isort might differ in behavior, but I had the same thought, it might just pick up the config from `pyproject`.